apps/openmw/mwrender/renderingmanager.cpp

```diff
@@ -506,7 +503,7 @@ namespace MWRender
         }
 
         if (!mSky->isEnabled() || !mSky->hasRain())
```

Comment:
If the weather is snow, this !hasRain() condition will fire and call clearRainRipples(), but things still happen to work because AlphaFader called in the Update phase runs later and resets the uniform to its expected (non zero) value. So from what I can tell, things will work but the code is odd - in its current form it could be reduced to clearRainRipples() every frame and still work. Could we move the clearing to Sky::setEnabled() maybe so that all changes to the uniform happen within sky.cpp?

Reply:
Oh yes I'll change it, I put this condition there because if I `coc`ed into an interior with water, the ripples stayed on, but I don't even know why I had the `!hasRain` there.